src/tinode.js

```diff
@@ -597,7 +661,7 @@ var Connection = function(host_, apiKey_, transport_, secure_, autoreconnect_) {
 
         log("Connecting to: " + url);
 
-        var conn = new WebSocket(url);
+        var conn = new WebSocketProvider(url);
```

Comment:
Where is WebSocketProvider defined?

Reply:
At the top

Reply:
There got to be a dependency on WebSocketProvider somewhere?

Reply:
I see `let WebSocketProvider` but that's just a variable declaration. There got be something like `require(...)` somewhere?

Reply:
We might add in the doc ... for Node JS call Tinode.setWebSocketProvider(require('ws'));